database/schema.sql

```diff
@@ -10,56 +13,53 @@
 -- 
 -- ---
 
-DROP TABLE IF EXISTS `Users`;
+DROP TABLE IF EXISTS users;
 		
-CREATE TABLE `Users` (
-  `id` INTEGER NULL AUTO_INCREMENT DEFAULT NULL,
-  `Name` MEDIUMTEXT(medium) NOT NULL DEFAULT 'NOT NULL',
-  `password` MEDIUMTEXT(medium) NULL DEFAULT 'NOT NULL',
-  `salt ` MEDIUMTEXT(medium) NULL,
-  PRIMARY KEY (`id`)
+CREATE TABLE users (
+  id INTEGER AUTO_INCREMENT PRIMARY KEY,
+  Name varchar(255) NOT NULL DEFAULT 'NOT NULL',
+  password varchar(255) NULL DEFAULT 'NOT NULL',
```

Comment:
Don't want them as 255. They'll be encrypted - Which I believe will be 64 characters